src/pages/edit_resource/EditResourcePage.tsx

```diff
@@ -255,7 +255,7 @@ const EditResourcePage: FC = () => {
   //triggers on uppy-events
   useEffect(() => {
     if (mainFileHandler) {
-      mainFileHandler.on('upload', () => {
+      mainFileHandler.on('upload-success', () => {
```

Comment:
Sikker på endringa her?
upload-eventen skjer når en enkeltt fil-opplasting starter. Upload-success er når den er ferdig med å laste opp alle filer.

Reply:
den var avglemt etter alt for mye fram og tilbake. bra du så den. :-)